test/utils/toMomentObject_spec.js

```diff
@@ -20,6 +20,10 @@ describe('toMomentObject', () => {
     expect(toMomentObject()).to.equal(null);
   });
 
+  it('output has time of 12PM', () => {
```

Comment:
it might be nice to test the actual conditions - like, that the thing resulting in the extra day being highlighted returns the correct result